packages/components/package.json

```diff
@@ -71,15 +75,18 @@
     "@kaizen/select": "^6.13.6",
     "@kaizen/split-button": "^1.3.33",
     "@kaizen/tabs": "^1.5.30",
-    "@kaizen/tailwind": "^0.5.1",
     "@kaizen/typography": "^2.3.6",
     "classnames": "^2.3.2"
   },
   "devDependencies": {
-    "@deanc/esbuild-plugin-postcss": "^1.0.2",
+    "@kaizen/tailwind": "^0.5.1",
```

Comment:
Moved this back to devDep cause the library doesn't require this to function, it only uses it at build time.